### ISSUE TITLE
Numpy bug/warning when calling MapieRegressor predict()

### ISSUE THREAD

**Issue by nilslacroix** [scikit-learn-contrib/MAPIE#164]
When calling predict on test set, this warning appears. This might be related to your aggregate_with_mask function
since it used np.median. Related issue see here: https://stackoverflow.com/questions/45786763/median-filter-produces-unexpected-result-on-fits-file

Error message on Linux:
```
anaconda3/envs/MASTER/lib/python3.10/site-packages/numpy/lib/function_base.py:4638: UserWarning: Warning: 'partition' will ignore the 'mask' of the MaskedArray.
  arr.partition(concatenate((virtual_indexes.ravel(), [-1])), axis=0)
```

**Comment by callmedrcom** [comment]
# i have two solution for this problem:
## 1. just ignore it :)
import warnings
warnings.filterwarnings('ignore')
## 2. delete the mask
the mask come from this:
ma.masked_invalid(lower_bounds) 
change it:
lower_bounds

may help~

**Comment by nilslacroix** [comment]
I ignored it but this seems to lead to other significant problems. I had a big dataset, where I had to split the prediction on many data folds, which worked fine. However for cv=10 my prediction looks like this:

![image](https://user-images.githubusercontent.com/56065345/172018528-5686b371-7975-4e31-90b0-5d38769796fa.png)

I assume 0 is the `single_estimator` attribute and 1-10 are the folds. Column `0` is the prediction and column `1` the interval. But why are they not aggregated, from my understanding the result should be a prediction on the dataset as a whole.

I think this is related to the `aggregate_with_mask` function but I do not know how to get the right prediction arrays here.
